src/nwb2bids/_tools/_cache_nwb.py

```diff
@@ -4,15 +4,21 @@
 
 import pynwb
 
-warnings.filterwarnings(action="ignore", message="No cached namespaces found in .+")
-warnings.filterwarnings(action="ignore", message="Ignoring cached namespace .+")
-warnings.filterwarnings(action="ignore", message="Stimulus description 'NA' for .+")
-warnings.filterwarnings(action="ignore", message="Use of icephys_filtering is deprecated and will be removed .+")
+ignored_pynwb_message_patterns = [
+    "No cached namespaces found in .+",
+    "Ignoring cached namespace .+",
+    "Ignoring the following cached .+",
+    "Stimulus description 'NA' for .+",
+    "Use of icephys_filtering is deprecated and will be removed .+",
+    "Date is missing timezone .+",
+]
+for message in ignored_pynwb_message_patterns:
+    warnings.filterwarnings(action="ignore", message=message)
 
 
 @functools.cache
 def cache_read_nwb(file_path: pathlib.Path) -> pynwb.NWBFile:
     """Cache the read operation per NWB file path to speed up repeated calls."""
-
-    nwbfile = pynwb.read_nwb(path=file_path)
+    resolved_path = file_path.resolve()  # Always resolve in case of symlinks (i.e., DataLad most likely)
```

Comment:
```suggestion
     # Always resolve in case of symlinks, mostly relevant to Windows (i.e., from DataLad most likely)
    resolved_path = file_path.resolve()
```